src/index.d.ts

```diff
@@ -12,14 +12,21 @@ declare namespace CarotteAmqp {
   export type ServiceName = string;
   export type ServiceVersion = number;
 
+  export type Logger = {
```

Comment:
Since the logger should come from Runtime, maybe to move into Runtime typing
Definitely have to work on typing anyway

Reply:
Ok to keep it there for me